.github/workflows/push.yml

```diff
@@ -35,6 +35,11 @@ jobs:
           cache-dependency-path: '**/pyproject.toml'
           python-version: ${{ matrix.pyVersion }}
 
+      - name: Setup Spark Remote
```

Comment:
We can change `test` in the Makefile instead so this won't be needed

Reply:
I would keep it separately for easier debugging

Reply:
agree but we should also run it as part of the `test` command

Reply:
I made some changes to Makefile I can do the other way name it as local test and keep the ci-test as test

Reply:
@mwojtyczka I still have 2 steps.

Reply:
i see, sounds good